subsys/bluetooth/host/mesh/adv.c

```diff
@@ -45,7 +45,11 @@
 #define ADV_INT_DEFAULT_MS 100
 #define ADV_INT_FAST_MS    20
 
+#ifdef CONFIG_BT_HOST_CRYPTO
```

Comment:
minor thing: we've tried to be consistent with `#if defined(CONFIG_...)` in the Bluetooth code, could you please do that? (benefit is that you can combine multiple `defined(...)` conditions for a single `#if` but not with `#ifdef`)